pkgs/applications/misc/spacefm/default.nix

```diff
@@ -4,7 +4,7 @@
 
 stdenv.mkDerivation rec {
   pname = "spacefm";
-  version = "1.0.6";
+  version = "1.0.6-1";
```

Comment:
Upstream has no 1.0.6-1 release. Please switch to unstable.

Reply:
You mean `version = "unstable"`?

Reply:
Searching through the code base, it seems the usual practice is using `unstable-DATE` or `VERSION-unstable-DATE`. I have updated the version using the latter format.